source/hands_on/sac.rst

```diff
@@ -270,7 +270,35 @@ Auto alpha strategy
+|                     |                 |                                                     |`config_link_ha <https:// |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|                     |                 |                                                     |DI-engine/blob/main/dizoo/| Baselines(9535)      |
+|Halfcheetah          |  12900          |.. image:: images/benchmark/halfcheetah_sac.png      |mujoco/config/halfcheetah_|                      |
+|                     |                 |                                                     |sac_default_config.py>`_  | Tianshou(12138)      |
+|(Halfcheetah-v3)     |                 |                                                     |                          |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_w <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Walker2d             |                 |                                                     |DI-engine/blob/main/dizoo/| Baselines(3863)      |
+|                     |  5172           |.. image:: images/benchmark/walker2d_sac.png         |mujoco/config/walker2d_   |                      |
+|(Walker2d-v2)        |                 |                                                     |sac_default_config.py>`_  | Tianshou(5007)       |
+|                     |                 |                                                     |                          |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_ho <https:// |                      |
+|                     |                 |                                                     |github.com/opendilab/     | Baselines(2325)      |
+|Hopper               |                 |                                                     |DI-engine/blob/main/dizoo/|                      |
+|                     |  3653           |.. image:: images/benchmark/hopper_sac.png           |mujoco/config/hopper_sac_ | Tianshou(3542)       |
+|(Hopper-v2)          |                 |                                                     |default_config.py>`_      |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
```

Comment:
shall we add Spinning-up as well for comparison. Also, it might be better to replace Baselines by Sb3

Reply:
added